test/IRGen/yield_once_enable_emit_type_malloc_coro_frame.sil

```diff
@@ -11,8 +11,8 @@ sil @marker : $(Builtin.Int32) -> ()
 // CHECK-SAME:  [[CORO_ATTRIBUTES:#[0-9]+]]
 sil @test_simple : $@yield_once () -> () {
 entry:
-  // CHECK-32: [[ID:%.*]] = call token (i32, i32, ptr, ptr, ptr, ptr, ...) @llvm.coro.id.retcon.once(i32 [[BUFFER_SIZE]], i32 [[BUFFER_ALIGN:4]], ptr %0, ptr @"$sIetA_TC", ptr @__swift_coroFrameAllocStub, ptr @free, i64 38223)
-  // CHECK-64: [[ID:%.*]] = call token (i32, i32, ptr, ptr, ptr, ptr, ...) @llvm.coro.id.retcon.once(i32 [[BUFFER_SIZE]], i32 [[BUFFER_ALIGN:8]], ptr %0, ptr @"$sIetA_TC", ptr @__swift_coroFrameAllocStub, ptr @free, i64 38223)
+  // CHECK-32: [[ID:%.*]] = call token (i32, i32, ptr, ptr, ptr, ptr, ...) @llvm.coro.id.retcon.once(i32 [[BUFFER_SIZE]], i32 [[BUFFER_ALIGN:4]], ptr %0, ptr @"$sIetA_TC", ptr @__swift_coroFrameAllocStub, ptr @free, i64 38224)
+  // CHECK-64: [[ID:%.*]] = call token (i32, i32, ptr, ptr, ptr, ptr, ...) @llvm.coro.id.retcon.once(i32 [[BUFFER_SIZE]], i32 [[BUFFER_ALIGN:8]], ptr %0, ptr @"$sIetA_TC", ptr @__swift_coroFrameAllocStub, ptr @free, i64 38224)
```

Comment:
@nate-chandler any idea what the +1 bump here would be from off hand?

Reply:
This value is computed by:

```
llvm::ConstantInt* IRGenFunction::getMallocTypeId() {                                                                                  
  return getDiscriminatorForString(IGM, CurFn->getName());                                                                             
}  
```

Reply:
Did this function change on rebranch?

Reply:
CC @t-rasmud

Reply:
Ah, yes.

This was:
```
static llvm::ConstantInt *getDiscriminatorForString(IRGenModule &IGM,
                                                    StringRef string) {
  uint64_t rawHash = clang::CodeGen::computeStableStringHash(string);
  return getDiscriminatorForHash(IGM, rawHash);
}
```

And is now:
```
static llvm::ConstantInt *getDiscriminatorForString(IRGenModule &IGM,
                                                    StringRef string) {
  uint64_t rawHash = llvm::getPointerAuthStableSipHash(string);
  return getDiscriminatorForHash(IGM, rawHash);
}
```

So `computeStableStringHash` != `getPointerAuthStableSipHash`. Any idea if this actually matters?

Reply:
Oh I see. `getPointerAuthStableSipHash` is doing the addition itself now (and returns a 16 bit int). All calls to `getDiscriminatorForString` can now just be `llvm::getPointerAuthStableSipHash(string)`. Mind doing that update @AnthonyLatsis?

Reply:
Nice catch!

Reply:
https://github.com/swiftlang/swift/pull/81213